package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "repository": "github:NES-Digital-Service/node-telemetry",
+  "name": "node-telemetry",
+  "description": "Provides standard Kubernetes probes and Prometheus metrics endpoint for Node.js apps.",
+  "scripts": {
+    "test": "jest --coverage"
+  },
```

Comment:
**question/thought:** There are publishing/naming/repository related peculiarities in https://github.com/NES-Digital-Service/ndp.audit-node/blob/main/package.json#L8 and I think they may be there for a purpose. It would be good modify the publish.yaml workflow as a temporary measure to try and publish once (package then can still be removed) to make sure publishing works before merging.